tox.ini

```diff
@@ -3,7 +3,6 @@
 # versions.
 
 [tox]
-requires = virtualenv<20
```

Comment:
why not `requires = virtualenv`? instead of `pip install -U virtualenv` above?

Reply:
It does not make any sense to have `requires = virtualenv`, it's already a requires. It might work as a _hack_, but that's it.
The proper fix is to remove `tox` from the base image so we always install the latest version.

Alternatively, we can do `requires = virtualenv<20,>=20.0.8`